pages/common/ccache.md

```diff
@@ -0,0 +1,21 @@
+# ccache
+
+> C/C++ compiler cache.
+> Note: ccache can also mock a compiler by changing the executable's name. Packages usually provide symlinks for common compilers at `/usr/lib/ccache/bin`. Prepend this directory to `$PATH` to automatically use ccache for these compilers.
```

Comment:
I tried to make it more concise, that line is kind of long.
```suggestion
> Note: packages usually provide symlinks for compilers in `/usr/lib/ccache/bin`. Prepend this directory to `$PATH` to automatically use `ccache` for them.
```

Reply:
Ok !